lib/linguist/languages.yml

```diff
@@ -5531,6 +5589,16 @@ mupad:
   - ".mu"
   ace_mode: text
   language_id: 416
+nanorc:
+  type: data
+  group: INI
+  extensions:
+  - ".nanorc"
+  filenames:
+  - nanorc
```

Comment:
I quickly went through the results from [your search query for that filename](https://github.com/search?p=96&q=filename%3Ananorc+NOT+nothack&type=Code), and there doesn't seem to be that many `nanorc` files, but there are a lot more results than we can view. Maybe we should estimate the number of `nanorc` files from the number found in the first 2K results?

Should we also add `.nanorc`? Same issue as for `nanorc` though.

Reply:
`nanorc`'s filename was mentioned in [`**nanorc**(5)`](https://linux.die.net/man/5/nanorc#adbottom), so it convinced me the usage wasn't a case of sloppy filenaming (`curlrc` also permits the dot to be omitted from user-configs, apparently...)

<pre><code>
<b>FILES</b>
       <b><u>SYSCONFDIR/nanorc</u></b>
              System-wide configuration file

       <b><u>~/.nanorc</u></b>
              Per-user configuration file
</code></pre>

> Should we also add `.nanorc`? Same issue as for `nanorc` though.

Ah, correct me if I'm wrong, but isn't a filename of `.foo` implicitly covered if there's an extension of `.foo` registered? Or am I getting that mixed up with the way Atom handles `fileTypes`? 😅

Reply:
Righto, I'll add it then. 👍

Reply:
Added, and OP updated with a link to [the sample I just added](https://github.com/baykovr/dotfiles/blob/e3c37157721e4f3dc5487fb5ba95ded963e3bac2/.nanorc). =)